app/validators/name_lenght_validator.rb

```diff
@@ -0,0 +1,9 @@
+class NameLenghtValidator < ActiveModel::EachValidator
```

Comment:
Why this validator is here? This is not a part of relation task. It should be on separate branch or on develop. Please remove it from here.